businesscentral-monitoring-app-secret.yaml

```diff
@@ -1,24 +0,0 @@
-kind: List
```

Comment:
see comment above.

Reply:
This service account and secret name was not being referenced at all in any template